pkg/insights/insightsreport/insightsreport.go

```diff
@@ -243,7 +243,8 @@ func (c *Controller) Run(ctx context.Context) {
 	c.StatusController.UpdateStatus(controllerstatus.Summary{Healthy: true})
 	klog.V(2).Info("Starting report retriever")
 	klog.V(2).Infof("Initial config: %v", c.configurator.Config())
-
+	clusterVersion, _ := c.client.GetClusterVersion()
+	insights.RecommendationCollector.SetClusterID(clusterVersion.Spec.ClusterID)
```

Comment:
We definitely need to use and handle the error, because otherwise this line can fail with nil panic.